### PR TITLE
To dts

### PR DESCRIPTION
**Pull request by sonofmun**

### CODE REVIEW

**Comment by sonofmun** [comment]
@PonteIneptique I don't need a review here yet. But the tests that are not passing now have to do with fetching annotations. And I am not sure precisely what has changed and where. I will keep researching, but if you have a tip where to look, I would appreciate it.

**Comment by sonofmun** [comment]
Perhaps a bit more explanation here. The source of the difference is from https://github.com/Capitains/MyCapytain/blob/3ff0f27e514f5895d4f83dbe294d4035a9e4dc53/MyCapytain/resources/texts/local/capitains/cts.py#L235. This used to return an empty list but now it returns an error.
But even if I catch the error here https://github.com/Capitains/flask-capitains-nemo/blob/2e87aa26a0444843f9814207031c4756a03097e9/flask_nemo/query/interface.py#L130 (instead of just checking if the list of reffs is empty), then the TestSimpleQuery tests e.g., https://github.com/Capitains/flask-capitains-nemo/blob/2e87aa26a0444843f9814207031c4756a03097e9/tests/test_query/test_interface.py#L125 still don't pass because the annotation is actually an empty list and thus annotation[0] throws an index error.

**Comment by PonteIneptique** [comment]
Matt, care to have a call this afternoon ?

**Comment by PonteIneptique** [comment]
If @balmas is available as well, that would be great.
I think we should drop annotations and other stuff, this might make your life easier.

**Comment by sonofmun** [comment]
Sure. I am available any time between 2pm and 5pm (so 7-10am eastern time).

**Comment by sonofmun** [comment]
FYI, with MyCapytain 3.0.0, the dts-draft-1 branch of Nautlius, and the to_DTS branch of flask-nemo, all of my unit tests work on formulae-capitains-nemo and everything seems to work locally as well.

**Comment by sonofmun** [comment]
I have solved the problem. The difference was that  MyCapytain.resources.texts.local.capitains.cts.getValidReff returned simply a list of the passages before 3.0.0 (e.g., ['1.1', '1.2'...]) whereas now it returns a list of `CtsReferenceSet`s, which are tuples. So that means that instead of a list of strings, what is returned is a list of tuples. And so that means that the lines that actually build the annotation list https://github.com/Capitains/flask-capitains-nemo/blob/2e87aa26a0444843f9814207031c4756a03097e9/flask_nemo/query/interface.py#L99-L106 would not match anything because `ref_in_range.intersection(annotation.target.expanded)` is never true (`ref_in_range` is made up of tuples and `annotation.target.expanded` of strings.
I will push a (very small) change here and then you @PonteIneptique can decide whether this makes sense or not.

**Comment by PonteIneptique** [comment]
I was actually having a look at it, and I am still not sure to understand why the first iteration on `self.four` yields `1.pr.1` as a target... Waiting for your commit then !

**Comment by PonteIneptique** [comment]
I think what you meant is getValidReff returns now CtsReferenceSet which are list of CtsReference which are tuple ?

**Comment by sonofmun** [comment]
That is correct. That is what I meant.

**Comment by PonteIneptique** [comment]
Have you tried to browse your current website using this version of nemo ?

**Comment by sonofmun** [comment]
I have. And everything works.

**Comment by PonteIneptique** [comment]
Ok, let me do some cosmetic change for one thing or two

**Comment by sonofmun** [comment]
It is still testing 3.4.5?

**Comment by PonteIneptique** [comment]
Yup and it should not. Let me do it ;)

**Comment by sonofmun** [comment]
OK.

**Comment by sonofmun** [comment]
Coverage goes down because the line that I left in query/interface.py for backward compatibility is not tested. Perhaps it can be removed since I have put MyCapytain>=3.0.0 in the requirements.txt?

**Comment by PonteIneptique** [comment]
I'll let you remove the line you think about removing, feel free to pull.

**Comment by PonteIneptique** [comment]
Oh and maybe bump the Minor version :')

**Comment by sonofmun** [comment]
Now you tell me.

**Comment by sonofmun** [comment]
Should it be 1.1 instead of 1.0.4?

**Comment by PonteIneptique** [comment]
Yup, 1.1.0 is what I meant with minor ;)

**Comment by sonofmun** [comment]
I am not sure about the terminology there.
BTW, you know that this will be a breaking change, right?

**Comment by sonofmun** [comment]
The description of Nemo in setup.py is "Flask Extension to browse CTS Repository". Should I change this? Perhaps, "Flask Extension to browse a CapiTainS-compliant Repository"?

**Comment by PonteIneptique** [comment]
Good catch. Yes :)

File: flask_nemo/__init__.py [hunk 1]
```diff
@@ -499,6 +499,7 @@ def expose_ancestors_or_children(self, member, collection, lang=None):
         x = {
             "id": member.id,
             "label": str(member.get_label(lang)),
+            "is_resource": isinstance(member, ResourceCollection),
```
Comment (PonteIneptique) [inline flask_nemo/__init__.py]: Can you add a test for that ?

Comment (PonteIneptique) [reply flask_nemo/__init__.py]: A plain text checking that on collection route, an Edition has the Browse link :)

File: CHANGES.mD [hunk 2]
```diff
@@ -1,6 +1,19 @@
 Change logs
 ===
 
+## 1.1.0 - 22/10/2019
```
Comment (PonteIneptique) [inline CHANGES.mD]: Shouldn't it be 2.0.0 ?

Comment (sonofmun) [reply CHANGES.mD]: Yes, it should be.

**Comment by PonteIneptique** [comment]
Looks good to me !

**Comment by sonofmun** [comment]
Should we PR to master as well and do a 2.0.0 release?

**Comment by PonteIneptique** [comment]
Yes :')

**Pull request merged**